a3/terraform/modules/cluster/gke/main.tf

```diff
@@ -63,11 +63,11 @@ module "dashboard" {
 module "resource_policy" {
   source = "../../common/resource_policy"
   for_each = {
-    for idx, node_pool in var.node_pools : "np-${idx}" => node_pool
+    for idx, node_pool in var.node_pools : idx => node_pool
     if node_pool.use_compact_placement_policy
   }
   project_id           = var.project_id
-  resource_policy_name = "${var.resource_prefix}-${each.key}"
+  resource_policy_name = "${var.resource_prefix}-policy-${each.key}"
```

Comment:
this resource will already be in the `google_compute_resource_policy` namespace

Reply:
addressed.